rerun_py/src/dataframe.rs

```diff
@@ -195,6 +196,108 @@ impl AnyComponentColumn {
     }
 }
 
+#[derive(FromPyObject)]
+enum IndexLike<'py> {
```

Comment:
`IndexLike` is a misleading name: this does not represent an index, it represents a series of index _values_. :thinking:

Reply:
Ahh, good call.